media/static-bundles.json

```diff
@@ -1074,6 +1080,17 @@
       ],
       "name": "firefox_whatsnew_69"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
+        "js/base/mozilla-fxa.js",
+        "js/base/mozilla-fxa-init.js",
+        "js/base/mozilla-monitor-button.js",
```

Comment:
Need to add `mozilla-monitor-button-init.js` here too?

Is it ok for the minotor button Js to be run from all versions of this page, or should it only be from the templates that feature a sign-in button?

Reply:
@alexgibson do you think that there is an excessive performance trade-off for including it on all versions? I didn't originally plan to make separate bundles, but I could.

Reply:
I wouldn't worry too much about performance, as the files are pretty small and will be minified. I was more wondering if redundant metrics flow requests could somehow skew some analytics somewhere on the FxA side? It may be fine though, I don't really know.

Reply:
oooh, excellent point. I am not sure either, so let's separate it just to be safe.